### PR TITLE
fixing issue with TF-graviton benchmark

### PR DESCRIPTION
**Pull request by xncqr**
*GitHub Issue #, if available:*

Note: If merging this PR should also close the associated Issue, please also add that Issue # to the Linked Issues section on the right. 

### Description
Fixing an issue with the Benchmark for TensorFlow. Testing on currently supported TensorFlow version and arches

### Tests run
[530e6c0](https://github.com/aws/deep-learning-containers/pull/2694/commits/530e6c0fe5d881d46d9615fc1ad63b675faefcac) TF 2.9 Graviton
* Benchmark Test

[863b04d](https://github.com/aws/deep-learning-containers/pull/2694/commits/863b04da43e8af27f0d2be2aa6e7fffa7a73dda8) TF 2.9 x86_64
* Benchmark Test

[dba0b40](https://github.com/aws/deep-learning-containers/pull/2694/commits/dba0b40eca462b11a75a2e30b1dace1e2e4b8061) TF 2.10 x86_64
* Benchmark Test

[959f25d](https://github.com/aws/deep-learning-containers/pull/2694/commits/959f25d0ad1a57b5ffe5ed1b9bff0f6315625d16) TF 2.11 x86_64
* Benchmark Test

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license. I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by ohadkatz — APPROVED** [review]
LGTM! 🚢

File: test/dlc_tests/benchmark/ec2/tensorflow/inference/test_performance_tensorflow_inference.py [hunk 1]
```diff
@@ -63,50 +63,49 @@ def test_performance_ec2_tensorflow_inference_graviton_cpu(
...
-
-    # Run performance inference command, display benchmark results to console
-
     if "graviton" in image_uri:
         # TF training binary is used that is compatible for graviton instance type
-
         ec2_connection.run(
             (
-                f"sudo apt install python3-pip"
+                f"/usr/bin/pip3 install --user --upgrade awscli boto3 grpcio"
```
Comment (arjkesh) [inline test/dlc_tests/benchmark/ec2/tensorflow/inference/test_performance_tensorflow_inference.py:83]: nit: maybe just leave a comment here saying the graviton AMI doesn't have these pkgs but the test needs it

**Review by arjkesh — APPROVED** [review]
LGTM. Left one nit comment but leaving approval

File: test/dlc_tests/benchmark/ec2/tensorflow/inference/test_performance_tensorflow_inference.py (hunk 1 above)
Comment (xncqr) [reply test/dlc_tests/benchmark/ec2/tensorflow/inference/test_performance_tensorflow_inference.py:83]: Understand the thought here. Am more looking at getting the AMIs consistent and removing most of this in the end. Better to look at refactoring this test on another PR as this PR is needed for patching of the DLC image. Thank you for the comment.

**Pull request merged**